src/routes/+layout.svelte

```
@@ -565,9 +565,15 @@
 	<link rel="stylesheet" type="text/css" href="/themes/rosepine-dawn.css" /> -->
 </svelte:head>
 
+<div style="width:100%;height:56px;background-color:#E61F23;padding-top:16px">
+	<img src={`${WEBUI_BASE_URL}/AccelBrain.png`} style="width:160px;height:24px; margin-left:45px;cursor:pointer" alt="logo" on:click={() => {
+		window.location.href = `${WEBUI_BASE_URL}/`;
+	}}/>
```

ESLint

[error] A11y: visible, non-interactive elements with an on:click event must be accompanied by a keyboard event handler. Consider whether an interactive element such as <button type="button"> or <a> might be more appropriate. See https://svelte.dev/docs/accessibility-warnings#a11y-click-events-have-key-events for more details.(a11y-click-events-have-key-events)

(svelte/valid-compile)